components/fieldlabel/stories/form.stories.js

```diff
@@ -1,36 +1,56 @@
+import { Template as Checkbox } from "@spectrum-css/checkbox/stories/template.js";
+import { Template as Fieldgroup } from "@spectrum-css/fieldgroup/stories/template.js";
 import { Template as Picker } from "@spectrum-css/picker/stories/template.js";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { Template as Stepper } from "@spectrum-css/stepper/stories/template.js";
 import { Template as TextField } from "@spectrum-css/textfield/stories/template.js";
+import { Template } from "./form.template.js";
 import pkgJson from "../package.json";
 import { FormGroup } from "./form.test.js";
 
 /**
- * The form component is used for aligning multiple inputs and field groups within a form.
+ * The form component is used for aligning multiple inputs and field groups within a form. It provides structure and spacing for your form fields.
  */
 export default {
 	title: "Form",
 	component: "Form",
 	argTypes: {
-		labelsAbove: {
-			name: "Labels above",
-			type: { name: "boolean" },
+		labelPosition: {
+			name: "Label position",
+			description: "Position of the field label in relation to the input.",
+			type: { name: "string" },
 			table: {
-				type: { summary: "boolean" },
+				type: { summary: "string" },
 				category: "Component",
 			},
-			control: "boolean",
+			control: "select",
+			options: ["top", "side"],
 		},
+		fieldLabelAlignment: {
+			name: "Field label alignment",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			control: "select",
+			options: ["left", "right"],
+			if: { arg: "labelPosition", eq: "side" },
+		},
+		items: { table: { disable: true } },
 	},
 	args: {
 		rootClass: "spectrum-Form",
-		labelsAbove: false,
+		labelPosition: "top",
 	},
 	parameters: {
 		packageJson: pkgJson,
 	},
 };
 
+/**
+ * A stacked layout with the labels above the form fields. The class `.spectrum-Form--labelPosition` is applied to the form itself. You do not need to apply a specific class to the field label because `.spectrum-FieldLabel--left` is applied to each to each [field label](/docs/components-field-label--docs) by default.
```

Comment:
According to the field label docs, top labels are default. https://spectrum.adobe.com/page/field-label/#Label-position